client/ayon_core/plugins/publish/collect_otio_subset_resources.py

```diff
@@ -174,9 +183,17 @@ def process(self, instance):
                     path, trimmed_media_range_h, metadata)
                 self.staging_dir, collection = collection_data
 
-            self.log.debug(collection)
-            repre = self._create_representation(
-                frame_start, frame_end, collection=collection)
+            if len(collection.indexes) > 1:
+                self.log.debug(collection)
+                repre = self._create_representation(
+                    frame_start, frame_end, collection=collection)
+            else:
+                filename, = tuple(collection)
```

Comment:
Isn't this the same?
```suggestion
                filename = collection[0]
```

Reply:
This is adjusted !